src/test/unit/utils/NotificationUtils.test.ts

```
@@ -0,0 +1,207 @@
+    sandbox = sinon.createSandbox();
+    clock = sinon.useFakeTimers();
+  });
+
+  afterEach(() => {
+    clock.restore();
+    sandbox.restore();
+    
+    // クリーンアップ
+    delete (global as any).document;
+    delete (global as any).window;
+    delete (global as any).HTMLElement;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)